### ISSUE TITLE
Uploading a stemcell of size 13GB gives error

### ISSUE THREAD

**Issue by ramyasg** [cloudfoundry/bosh-lite#437]
We have a stemcell which we built from a vmdk file and has a file size of 13 GB. 
When we tried to upload this to our director, obtained this error - 

> Uploading stemcell file:
>   Director responded with non-successful status code '413' response '<html>
> <head><title>413 Request Entity Too Large</title></head>
> <body bgcolor="white">
> <center><h1>413 Request Entity Too Large</h1></center>
> <hr><center>nginx</center>
> </body>
> </html>
> '
> 
> Exit code 1

Read through the bosh documentation that the size limit is 10GB only. Is there any way to increase this limit. Any one know how to modify nginx settings to increase this size.

**Comment by grafpoo** [comment]
yes, it's a setting for nginx that you can modify. i've found (and
adjusted) it for regular cloud-foundry/bosh

**Comment by ramyasg** [comment]
@grafpoo  It would be a great help if you can point us to any link where we can get more details about tweaking the settings for nginx.

**Comment by dpb587-pivotal** [comment]
Hi - typically you should be using the stemcells from bosh.io for BOSH Lite / Warden. The VMDK stemcells are not supported in bosh-lite, and I don't think you'll be able to deploy to them after uploading. I'd strongly recommend using the stemcells from bosh.io to get started.

Do you mind describing why you're needing to create stemcells? We don't recommend this because they diverge from our officially maintained images with our base security and system settings that releases expect. Any customizations to VMs should be happening through BOSH releases which helps keep the stemcells relatively slim (much, much less than 13GB).

**Comment by ramyasg** [comment]
Hi, Thank you for the reply. We have experimented with the default stemcell that was available with bosh.io. However our research deals with usage of Windows 2012 R2 on VSphere. Hence the need for creating custom stemcell. 
If someone can point out to a link or blog where in we can modify the file size limit and also on how to login to the Director, it will help us in proceeding with our exploration as currently we are stuck in this step.
